packages/plugin-core/src/components/pods/PodControls.ts

```diff
@@ -364,6 +434,12 @@ export class PodUIControls {
       case PodExportScope.Selection:
         return "Exports the current contents of the selected portion of text in the open note editor";
 
+      case PodExportScope.Lookup:
+        return "Exports all notes selected by lookup";
```

Comment:
nit: How about "Prompts user to select note(s) for export"?

Basically, to give a better indication that this will allow/require for user input, whereas the other options don't really do that.